src/Config.elm

```diff
@@ -0,0 +1,14 @@
+module Config exposing (..)
+
+import Dict exposing (Dict)
+
+type alias Config = 
+    Dict String Bool
+
+    
+defConfig : Config
```

Comment:
`defaultConfig`, the `def` abbreviation is confusing (default? definition? define? definitive?)